src/DIRAC/__init__.py

```diff
@@ -190,6 +190,9 @@ def _computeRootPath(rootPath):
 else:
     pythonPath = os.path.realpath(__path__[0])
     rootPath = os.path.dirname(pythonPath)
+# Allow rootPath detection to be overridden
+if "DIRACTOP" in os.environ:
+    rootPath = os.environ["DIRACTOP"]
```

Comment:
If we end up going with this solution I think the variable should be something like `DIRAC_ROOT_PATH` or `DIRAC_INSTALL_ROOT`.